src/OrbitQt/include/OrbitQt/orbitlivefunctions.h

```diff
@@ -13,6 +13,7 @@
 #include <QString>
 #include <QVBoxLayout>
 #include <QWidget>
+#include <cstdint>
```

Comment:
We already include stdint.h above. Also, in orbit we prefer stdint.h over cstdint, as the latter guarantied to provide std::uint64_t, not uint64_t.

Reply:
thanks for the reminder